src/containers/LocationPage/LocationPage.jsx

```diff
@@ -26,7 +26,7 @@ import {
   MetricSelector,
   TimeAggregationSelector,
   StatusWrapper,
-  // IspSelectWithIncidents,
+  IspSelect,
```

Comment:
Can you delete all the changes from this file. This was just to get the incident ISP dropdown on your machine

Reply:
Deleted changes in LocationPage file